deployments/warp_routes/USDC/eclipse/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "USDC",
+  "symbol": "USDC",
+  "description": "Warp Route USDC on Eclipse",
+  "image": "https://github.com/spothq/cryptocurrency-icons/blob/1a63530be6e374711a8554f31b17e4cb92c25fa5/128/icon/usdc.png",
```

should it be <raw.....> as done for SOL?
https://github.com/hyperlane-xyz/hyperlane-registry/blob/4ee77e9bf90eff2f99d66b4c7b941fe6d3da9612/deployments/warp_routes/SOL/eclipse/metadata.json#L5

```suggestion
  "image": "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/128/icon/usdc.png",
```

---

great catch, pushed with the permalink